src/storage/query/QueryBaseProcessor.inl

```diff
@@ -98,6 +100,8 @@ cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkAndBuildContexts(const REQ&
                 } else {
                     tagContexts_[it->second].props_.emplace_back(std::move(prop));
                 }
+
+                buildTagOrEdgeTTLInfo(schema.get(), true, tagId, 0);
```

Comment:
Maybe should put the method after the for loop.  We don't need to do it for every props.

Reply:
merge  `buildTagOrEdgeTTLInfo `and  build `RespSchema `  to  `buildTTLInfoAndRespSchema`